test/integration/tarantool_role_test.lua

```diff
@@ -0,0 +1,155 @@
+local t = require('luatest')
+local fio = require('fio')
+
+local helpers = require('test.helper')
+local Server = require('test.integration.server')
+
+local g = t.group('tarantool_role_integration_test')
+local tmpdir = fio.tempdir()
+
+g.before_all(function (cg)
+    t.skip_if(not helpers.tarantool_role_is_supported(),
+    'Tarantool role is supported only for Tarantool starting from v3.0.0')
+
+    cg.router = Server:new({
+        config_file = fio.abspath(fio.pathjoin('test', 'integration', 'simple_app', 'config.yaml')),
+        env = {rocks_dir = fio.abspath('.')},
+        chdir = fio.pathjoin(tmpdir, 'simple_app'),
+        alias = 'master',
+        workdir = fio.pathjoin(tmpdir, 'simple_app'),
+    })
+
+end)
+
+g.before_each(function(cg)
+    fio.mktree(cg.router.workdir)
+    fio.copytree(fio.abspath('./.rocks'), fio.pathjoin(cg.router.workdir, '.rocks'))
```

Comment:
Why is it required?

Reply:
tarantool is not able to find roles.expirationd in test setup w/o it.

Reply:
Why not? I really don't understand the problem.

Reply:
Either we provide code (both `expirationd` and `roles` folders) through source code or through `.rocks`.

In both cases, I think it's better to simply provide `LUA_PATH` in case you start a server in a separate environment.